### PR TITLE
Update surf_mesa.cfg

### PR DESCRIPTION
**Pull request by Kyli3Boi**
Added filter to remove map sounds, which was spamming the console with multiple errors

### CODE REVIEW

File: addons/stripper/maps/surf_mesa.cfg [hunk 1]
```diff
@@ -137,3 +137,8 @@ add:
   "classname" "env_laser"
   "hammerid" "3875"
 }
+
+filter:
```
Comment (sneak-it) [inline addons/stripper/maps/surf_mesa.cfg:141]: Just nitpicking, you can add this filter to the top of the file after the env_sun removal for cleanliness.

Like so:
```
filter:
{
  "classname" "env_sun"
}
{
  "classname" "env_soundscape"
}
```

**Comment by Kyli3Boi** [comment]
Sorry for that Disaster, im still not confident in how github works.

I have taken your suggestion and amended

**Comment by Kyli3Boi** [comment]
If there is a way i can clean up all those commits please let me know

**Comment by sneak-it** [comment]
Looks great, thanks for your contribution!

**Pull request merged**